soco/core.py

```diff
@@ -1388,31 +1388,28 @@ def add_multiple_to_queue(self, items, container=None):
             items (list): A sequence of items to the be added to the queue
```

Comment:
So I think I suggested some of the formulation that describes `items`. The reason I suggested "sequence" is that the original version of this code really only required a sequence, since the only that you were doing with it was to iterate over it. This meant e.g. that it would have worked fine with a set. But now you also need indexing (slicing really), which means that you should probably either change this formulation to explicitly say "list" or change the code to turn items into a list when it enters.